src/consensus/pbft/pbfttypes.h

```diff
@@ -91,6 +96,14 @@ namespace pbft
               ccf::Store::Tx tx(version);
               auto pp_view = tx.get_view(pbft_pre_prepares_map);
               pp_view->put(0, pp);
+              auto sig_view = tx.get_view(signatures);
+              crypto::Sha256Hash sig_hash(
+                {CBuffer{pp.digest_sig.data(), pp.digest_sig.size()}});
+              // we only need the hash of the pre prepare as the pre-prepare
+              // which contains the root of the merkle tree will be stored in
+              // the pbft_pre_prepares map
+              ccf::Signature sig_value(sig_hash);
+              sig_view->put(0, sig_value);
```

Comment:
I am confused by the logic here. Why not just include the signature what property does the hash give us? Especially since we need to pay for doing the extra hashing?

Reply:
The reason that is is getting added is for the members to be able to ack over the root of the latest signature. In pbft since the signature table is not being persisted to the ledger, we don't really need to store the actual signature or the other information at this point (view, seqno) since it is (1) stored in the pre-prepares table and (2) not used anywhere

Reply:
discussed offline - I think this is fine

Reply:
Replaced the hash over the pre-prepare signature with the merkle root of the pre-prepare